app/CocoaPods/CPBrownVisualEffectsView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CPBrownVisualEffectsView.swift
+//  CocoaPods
+//
+//  Created by Orta on 11/27/15.
+//  Copyright © 2015 CocoaPods. All rights reserved.
+//
+
```

Comment:
I'll make a separate PR after this removing all of them from all the m & h too, it'd get super ugly here too.